rasa_core/server.py

```diff
@@ -19,8 +19,13 @@
 from rasa_core.events import Event
 from rasa_core.policies import PolicyEnsemble
 from rasa_core.test import test
-from rasa_core.trackers import DialogueStateTracker, EventVerbosity
+from rasa_core.trackers import EventVerbosity
 from rasa_core.utils import dump_obj_as_str_to_file
+from rasa_core.agent import load_agent
+from rasa_core.trackers import DialogueStateTracker
```

Comment:
Imports from same python file can be merged to be in same line, for e.g., from rasa_core.trackers import EventVerbosity, DialogueStateTracker